website_and_docs/content/documentation/webdriver/elements/finders.zh-cn.md

```diff
@@ -279,10 +261,9 @@ fun main() {
   {{< /tab >}}
 {{< /tabpane >}}
 
-## Find Elements From Element
+## 从单个元素中查找多个元素
 
-It is used to find the list of matching child WebElements within the context of parent element.
-To achieve this, the parent WebElement is chained with 'findElements' to access child elements
+它用于在父元素上下文内查找匹配子元素的列表，实现这个，父元素使用`findElements`访问子元素。
```

Comment:
you can try better on "To achieve this, the parent WebElement is chained with 'findElements' to access child elements"